### PR TITLE
Clarify the duplicates resolver dialog actions

### LINKED ISSUE #9056: Clarify button "Import and keep old entry"
Refs https://github.com/JabRef/jabref/issues/6190

The button text "Import and keep old entry" does not make it crystal clear, what happens.

My use case:

1. Copy 120 entries (which are no duplicates) from a bib file
2. Paste these entries in a new file
3. The merge entries dialog pops up
4. I want to keep both entries (and not creating a new one)

grafik

### PR DESCRIPTION
**Pull request by HoussemNasri**
Fixes #9056 

**TODO**
- [x] Add a smaller help button.
- [x] Handle the `Automatically remove exact duplicates` button.
- [x] CHANGELOG.md

### ImportEntriesDialog *(triggered when importing another library into the current library)* and ImportHandler *(triggered when pasting an entry from the clipboard)*

image

### The default DuplicateResolverDialog *(triggered by choosing 'Find duplicates' from the menu)*

Capture1

### Help button and Remove exact duplicates button visible
remove-exact-visible

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/duplicationFinder/DuplicateResolverDialog.java [hunk 1]
```diff
@@ -131,8 +122,8 @@ private void init(BibEntry one, BibEntry two, DuplicateResolverType type) {
         });
 
         getDialogPane().setContent(borderPane);
-        Button helpButton = (Button) this.getDialogPane().lookupButton(help);
-        helpButton.setOnAction(evt -> helpCommand.execute());
+/*        Button helpButton = (Button) this.getDialogPane().lookupButton(help);
+        helpButton.setOnAction(evt -> helpCommand.execute());*/
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/duplicationFinder/DuplicateResolverDialog.java:126]: ?

**Review by calixtus — APPROVED** [review]
LGTM except one comment left...

File: src/main/java/org/jabref/gui/externalfiles/ImportHandler.java [hunk 2]
```diff
@@ -195,9 +195,9 @@ public void importEntryWithDuplicateCheck(BibDatabaseContext bibDatabaseContext,
 
         Optional<BibEntry> existingDuplicateInLibrary = new DuplicateCheck(Globals.entryTypesManager).containsDuplicate(bibDatabaseContext.getDatabase(), entryToInsert, bibDatabaseContext.getMode());
         if (existingDuplicateInLibrary.isPresent()) {
-            DuplicateResolverDialog dialog = new DuplicateResolverDialog(existingDuplicateInLibrary.get(), entryToInsert, DuplicateResolverDialog.DuplicateResolverType.IMPORT_CHECK, bibDatabaseContext, stateManager, dialogService);
+            DuplicateResolverDialog dialog = new DuplicateResolverDialog(existingDuplicateInLibrary.get(), entryToInsert, DuplicateResolverDialog.DuplicateResolverType.IMPORT_CHECK, bibDatabaseContext, stateManager, dialogService, preferencesService);
             switch (dialogService.showCustomDialogAndWait(dialog).orElse(DuplicateResolverDialog.DuplicateResolverResult.BREAK)) {
-                case KEEP_LEFT:
+                case KEEP_RIGHT:
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/externalfiles/ImportHandler.java:200]: Left and right have changed?

File: src/main/java/org/jabref/gui/duplicationFinder/DuplicateResolverDialog.java [hunk 3]
```diff
@@ -130,9 +129,11 @@ private void init(BibEntry one, BibEntry two, DuplicateResolverType type) {
             return null;
         });
 
+        HelpAction helpCommand = new HelpAction(HelpFile.FIND_DUPLICATES, dialogService);
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/duplicationFinder/DuplicateResolverDialog.java:132]: @ThiloteE  Help Button

File: src/main/java/org/jabref/gui/externalfiles/ImportHandler.java (hunk 2 above)
Comment (HoussemNasri) [reply src/main/java/org/jabref/gui/externalfiles/ImportHandler.java:200]: In the old UI, the order of the buttons was inverted. It was left:'Remove old entry' and right:'Remove from import'. By investing left and right, left will remove from import which means it keeps the old entry and right will remove old entry which is equivalent to keeping the from import entry.
I know it is a little confusing 😅 you can try it to make sure it works.

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**